server/container/guice/memory-guice/src/test/java/org/apache/james/GuiceJamesServerTest.java

```diff
@@ -35,6 +91,15 @@
             .disableAutoStart()
             .build();
 
+        private GuiceJamesServer serverWithStartUpCheck;
```

Comment:
I would rather have these tests in a dedicated class rather than adding a second server....

Reply:
okay